sklearn/utils/tests/test_validation.py

```diff
@@ -31,6 +33,7 @@
     check_is_fitted,
     check_consistent_length,
     assert_all_finite,
+    check_memory
```

Comment:
Just as a note for you future PR:
We only make put absolute path in the test file like here :)